### PR TITLE
automate publish to github package

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: scripts/publish-to-github.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+echo "@dimensiondev:registry=https://npm.pkg.github.com/DimensionDev" > "$HOME/.npmrc"
+echo "//npm.pkg.github.com/:_authToken=${NODE_AUTH_TOKEN}" >> "$HOME/.npmrc"
```
Comment (Jack-Works) [inline scripts/publish-to-github.sh:3]: Does the token configured to expose to the `master` branch only?

File: package.json [hunk 2]
```diff
@@ -1,39 +1,40 @@
 {
-  "name": "@holoflows/kit",
-  "version": "0.7.0",
-  "module": "./es/index.js",
-  "main": "./umd/index.js",
-  "typings": "./es/",
-  "repository": "https://github.com/DimensionDev/Holoflows-Kit",
+  "name": "@dimensiondev/holoflows-kit",
```
Comment (Jack-Works) [inline package.json:2]: This package is also published to the `npm` registry. How to deal with this?

File: scripts/publish-to-github.sh (hunk 1 above)
Comment (septs) [reply scripts/publish-to-github.sh:3]: https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            - uses: actions/checkout@v2
+            - name: Use Node.js 12.x
+              uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+            - uses: jgierer12/npm-auth-gpr@v1
+              with:
+                  token: ${{ secrets.GITHUB_TOKEN }}
+            - name: Install and Test
+              run: yarn install
```
Comment (Jack-Works) [inline .github/workflows/nodejs.yml:22]: I see the install part, but where is the build & test part?

Comment (Jack-Works) [reply .github/workflows/nodejs.yml:22]: build part?

File: README.md [hunk 4]
```diff
@@ -10,6 +10,15 @@ Toolkit for developing Augmented Virtuality.
 
 [API Documentation](./api-documents/kit.md)
 
+## Usage unstable version
```
Comment (Jack-Works) [inline README.md:13]: Usage is a noun, not a verb 🤔 but I'm not blocking here

**Pull request merged**